file.txt

```diff
@@ -1 +1,2 @@
 brandon text
+added line
```

Comment:
are you sure you want to do this? our spec says all lines should follow proper capitalization